src/emscatter.jl

```diff
@@ -0,0 +1,296 @@
+# In this tutorial, we will learn:
+# 
+#   * How to formulate the weak form for time-harmonic electromagnetic problem
+#   * How to implement perfectly matched layers (PML)
+#   * How does periodic boundary condition work in Gridap
+# 
+# # Problem statement
+# 
+# The problem that we are trying to solve is the plane wave (TM-polarized $H_{inc}$) scattering of a cylinder dielectric (of radius $R$ and permittivity $\varepsilon$), as illustrated below. The computational cell is of height $H$ and length $L$, and we designate a perfectly matched layer (PML) thickness of $d_{pml}$.
```

Comment:
I would describe this as a scalar wave problem, and then say it corresponds (for example) to Hz-polarized electromagnetic waves scattering off a dielectric cylinder.

I wouldn't use the "TM" terminology since it is too ambiguous.